### PR TITLE
Add Cytomining ecosystem visualizations, configure GitHub Pages, and add to GitHub data

### PR DESCRIPTION
**Pull request by d33bs**
This PR makes a number of changes largely focused on increasing the amount of data extracted, creating visualizations, and configuring GitHub pages content for with the visualizations.

Thank you for any feedback you may have! Please don't hesitate to let me know if you have any questions.

### CODE REVIEW

File: notebooks/cytomining-ecosystem/visualize-landscape.py [hunk 1]
```diff
@@ -0,0 +1,353 @@
...
+
+# +
+# create a duration for relative comparisons below
+df_projects["Duration Created to Now"] = current_datetime - df_projects["Date Created"]
+
+# create a years count for project time duration
+df_projects["Duration Created to Now in Years"] = (
+    df_projects["Duration Created to Now"].dt.days / 365
+)
+# -
```
Comment (gwaybio) [inline notebooks/cytomining-ecosystem/visualize-landscape.py]: You may also consider calculating a log transformed axis for star count and visualizaing this instead

**Review by gwaybio — APPROVED** [review]
🤩

File: notebooks/cytomining-ecosystem/visualize-landscape.py (hunk 1 above)
Comment (d33bs) [reply notebooks/cytomining-ecosystem/visualize-landscape.py]: Ah great point, thanks! I'll add this in.

**Comment by d33bs** [comment]
Thanks @gwaybio! Going to merge this in.

**Pull request merged**